src/credentials/attestation_verifier/DeviceAttestationVerifier.h

```diff
@@ -386,6 +386,9 @@ class DeviceAttestationVerifier
                                                            const Crypto::P256PublicKey & dacPublicKey,
                                                            const ByteSpan & csrNonce) = 0;
 
+    virtual void ValidateDACChainRevocationStatus(const AttestationInfo & info,
```

Comment:
This is public API, which needs API documentation.

This is also going to break all existing API clients that implement a custom attestation verifier, right?  Is that the intent?

Reply:
Addressed in latest commit